chunky/src/java/se/llbit/chunky/renderer/scene/AsynchronousSceneManager.java

```diff
@@ -86,6 +86,13 @@ public void setOnChunksLoaded(Runnable onChunksLoaded) {
       }
     } catch (InterruptedException ignored) {
       // Interrupted.
+    } catch (Throwable e) {
+      if (e instanceof OutOfMemoryError) {
+        Log.error("Chunky has run out of memory! Increase the memory given to Chunky in the launcher.");
+      }
```

Comment:
Should we only print one error (= only one tab) per error?

Reply:
A stacktrace is probably helpful for advanced users. For example, an OOM in octree = use octree plugin. OOM in bvh = use bvh plugin.

Reply:
it sure is, but maybe put the stack trace into the same message so that it's still a single tab?